python/test-r_bp.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python
```

Comment:
Shouldn't we use `/usr/bin/env python3` for now? Just in case.

Reply:
Sure, will fix.